### DISCUSSION TITLE
DPL wie wird gesetzt.

### DISCUSSION THREAD

**Discussion by rojanet** [tbnobody/OpenDTU#2238] [General]
Hallo Ich hätte da mal eine frage.

wie werden bei eingeschaltetem DPL die werte Leistung gesetzt.

also nonpersistent oder als persistent.

vielen Dank.

**Comment by stefan123t** [comment]
@rojanet das ist keine Diskussion sondern eine einfache Frage:

Die regelmäßig geänderten Werte sollten idR nur als temporäre (non-persistent) ActivePowerLimit (APL) gesetzt werden. Ein ständig wechselndes APL als persistent zu speichern verschleißt nur das EEPROM des Inverters an der dieses permanente Limit gespeichert wird.
